rs/sns/governance/canister/canister.rs

```diff
@@ -68,6 +70,15 @@ const STABLE_MEM_BUFFER_SIZE: u32 = 100 * 1024 * 1024; // 100MiB
 
 static mut GOVERNANCE: Option<Governance> = None;
 
+thread_local! {
+    static TIMER_ID: RefCell<Option<TimerId>> = RefCell::new(Default::default());
+}
+
+/// This guarantees that timers cannot be restarted more often than once every 300 intervals.
+const RESET_TIMERS_COOL_DOWN_INTERVAL: Duration = Duration::from_secs(600);
+
+const RUN_PERIODIC_TASKS_INTERVAL: Duration = Duration::from_secs(2);
```

Comment:
To get any real benefit from timers, we may want to set it a little higher than this?  Currently we're running heartbeat a bit more than 1x/second I think, and timers are a little bit more expensive.  So I don't know if we're going to see a huge lift from just making it every 2 seconds.  But we can monitor and find out I guess

Reply:
Thanks for the feedback. Setting it to 5 seconds for now.